client/src/components/loginRegisterModal.js

```diff
@@ -82,96 +80,97 @@ export default function LoginRegisterModal(props) {
 				password: ''
 			});
 			setDialogOpen(false);
-			return res
+			return res;
 		} catch (err) {
-			const errors = err.response.data.errors;//server-end doesn't return msg
-			setErrorMsg("Logging in failed. Try again.");
+			const errors = err.response.data.errors; //server-end doesn't return msg
+			setErrorMsg('Logging in failed. Try again.');
 			setTimeout(() => {
 				setErrorMsg('');
 			}, 4000);
 		}
-		return null
+		return null;
 	};
-	
 
 	useEffect(() => {
-		if (asyncStart){
-			loginRegister(name, email, password, loginActive)
-			.then(res=>{
+		if (asyncStart) {
+			loginRegister(name, email, password, loginActive).then((res) => {
 				if (res && res != null && res.data != null) {
 					setUserData(res.data);
 					authContext.handleLogin(res.data);
 					console.log(res);
-				}else{
-					console.log("error: fetching user data failed.");
-				};
+				} else {
+					console.log('error: fetching user data failed.');
+				}
 			});
 			setAsyncStart(false);
 		}
-		console.log("test login/useEffect");
+		console.log('test login/useEffect');
```

Comment:
we can take this and the previous console.logs out.

Reply:
Will do!